components/containers/Admin/Organisations/OrganisationSettings.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment } from "react";
+import { useTranslation } from "next-i18next";
+import Head from "next/head";
+import { Organisation } from "@lib/types";
+import { DeleteButton } from "../../../forms/Button/DeleteButton";
```

Comment:
This can be added to the forms package index in `/.components/forms/index.ts` for easier imports.